packages/babel-parser/test/fixtures/core/escape-keyword/invalid/options.json

```diff
@@ -0,0 +1,3 @@
+{
+  "throws": "Unexpected keyword 'break' (2:2)"
```

Comment:
Shouldn't the error be `Escape sequence in keyword break`?

Reply:
Well, both the errors would be ok since here `break` is parsed as a keyword.